_layouts/documentation.html

```diff
@@ -3,35 +3,36 @@
 ---
 
 <main class="usa-layout-docs usa-section" id="main-content">
-  <div class="grid-container">
+  <div class="grid-container grid-container__full-width">
```

Comment:
Thanks for pointing this out, I agree we should width constrain it at the very least.. I'm wondering if we want to keep it left aligned rather than center though. @anniehirshman-gsa  what do you think?

Reply:
I went ahead and pushed the center aligned option to the PR so you can see what it would look like centered (its actually the easier option compared to left-aligning I think)

Reply:
In the Figma I was going for left/right aligned, width constrained (header max width 100rem). And I agree, it'll help a lot for the hero text to be left-aligned with the left edge of the main content area.

Feel free to send me a preview link and I'll take a closer look and review!